app/views/devise/shared/_auth_nav.html.slim

```diff
@@ -0,0 +1,10 @@
+.clearfix.mxn1.mb1
+  - login_page = controller_name == 'sessions'
+  .sm-col.sm-col-6.px1.mb1
+    = link_to t('upaya.links.sign_up'), new_user_registration_path, \
+      class: 'btn btn-primary border-box col-12 center ' \
+             "#{login_page ? 'bg-gray' : 'bg-navy'}"
+  .sm-col.sm-col-6.px1.mb1
+    = link_to t('upaya.links.sign_in'), new_user_session_path, \
+      class: 'btn btn-primary border-box col-12 center ' \
+             "#{login_page ? 'bg-navy' : 'bg-gray'}"
```

Comment:
that said, you are the rails guru :), so if you'd prefer this path, i'm happy to switch to it